src/mudlet-lua/lua/Other.lua

```diff
@@ -425,8 +425,9 @@ local speedwalkShow
 --- Stops a speedwalk and clears the walklist
 function stopSpeedwalk()
   local active = pauseSpeedwalk()
-  if active then 
+  if active then
     speedwalkList = {}
+    raiseEvent("sysSpeedwalkStopped")
```

Comment:
Thinking in terms of design, shouldn't we also have events for speedwalk started and speedwalk unpaused?

Reply:
I suppose that makes sense. I certainly can't think of a good reason -not- to include events for those.